pep-9999.rst

```diff
@@ -26,20 +28,36 @@ formalized the governance of tools included under the PyPA
 umbrella, this governance does not extend to setting long-term goals 
 for PyPA tools. In order to ensure PyPA tools remain relevant in the 
 future and cater to user requirements, it is important to set a vision 
-and strategy for Python Packaging and PyPA tools.
+and strategy for Python Packaging and PyPA tools. It is hoped that 
+the council will focus on the Packaging ecosystem, define the 
+vision and work with the community to develop the strategy. 
+Attempts at defining the vision and strategy have failed so far.
 
 The main motivation for this PEP is to create a body with the necessary 
 powers to define the short-term and long-term goals for Python Packaging 
 and to support the community in setting the strategy to achieve the goals. This 
 governing body will represent PyPA tools when working with any other tool or group. 
 
+As such, the mission of the initial council will be-
+
+* Define the short-term and long-term vision and strategy for Python Packaging
+* Approve PEPs that support the Packaging strategy
+* Identify solutions to few (1-2) problems identified in the 
+[Python Packaging survey](https://drive.google.com/file/d/1U5d5SiXLVkzDpS0i1dJIA4Hu5Qg704T9/view) and marshall resources to implement the solutions
```

Comment:
RST link rather than Markdown, and a typo:
```suggestion
`Python Packaging survey <https://drive.google.com/file/d/1U5d5SiXLVkzDpS0i1dJIA4Hu5Qg704T9/view>`__
and marshal resources to implement the solutions
```